tracks/ruby/exercises/two-fer/mentoring.md

```diff
@@ -1,34 +1,53 @@
+
+```ruby
+module TwoFer
+  module_function
+
+  def two_fer(name = 'you')
+    format('One for %<name>s, one for me.', name: name)
+  end
+end
+```
+Solutions can also use string templates with `format` or `%`:
+
+```ruby
+class TwoFer
+
+  class << self
+    def two_fer(name = 'you')
+      'One for %s, one for me.' % name
+    end
+  end
+end
+```
 
 ### Common suggestions
-- Suggest using a default value instead of any form of conditionals. 
-- People often set the default of `name` to `nil`. Ask if there is something they could do with the default value to avoid the conditional and make their code simpler.
-- Suggest to remove `return`
-- Use `def` with parentheses, because of the parameter.
-- Check if the BookKeeper module is present and the comment removed.
-- Some people need help with running the tests. 
-- Check indentation (convention is 2-space indentation).
+- Use a default value instead of any form of conditionals.
+- When a student set the default of `name` to `nil`: Ask if there is something they could do with the default value to avoid the conditional and make their code simpler.
+- Some people need help with running the tests.
+- Style suggestions based on community conventions:
+  - Remove explicit `return` ([no-explicit-return](https://github.com/rubocop-hq/ruby-style-guide#no-explicit-return))
+  - Use `def` with parentheses, because of the [parameter (method-parens)](https://github.com/rubocop-hq/ruby-style-guide#method-parens).
+  - Use indentation of [2-spaces (spaces-indentation)](https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation)).
 
 ### Talking points
-- Implicit returns
+- [Implicit returns](https://franzejr.github.io/best-ruby/idiomatic_ruby/implicit_return.html)
 - Default values
-- String interpolation (http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html)
-- Style preferences. This exercise is a good opportunity to talk about style conventions like indentation, parameter parenthesis in method declarations and removing redundant comments. 
-https://github.com/rubocop-hq/ruby-style-guide
-Given the place in the curriculum, it may be worth to not going to deep, and to not addressing points that are controversial or personal preference. 
+- String interpolation (http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html), but a student may also use string templates via `format` or `%`. You may mention the alternative technique (templates when interpolation is used, and vice versa), always prefer either over string concatination.
+- Style preferences based on [community style guide via rubocop](https://github.com/rubocop-hq/ruby-style-guide): This exercise is a good opportunity to talk about style conventions like indentation, parameter parenthesis in method declarations and removing redundant comments. Given the place in the curriculum, it may be worth to not going to deep, and to not addressing points that are controversial or personal preference.
```

Comment:
"may be worth not going too deep"  Remove the first "to" and correct the second *to* spelling to _too_.

Reply:
Copied from a different note; will PR the fix there too!